pubspec.yaml

```diff
@@ -1,5 +1,5 @@
 name: nyxx
-version: 5.0.0
+version: 5.0.1
```

Comment:
Need to bump version in constants.dart as well

Reply:
Oh, it already is. Missed that.

Reply:
it is